src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -103,7 +104,9 @@ def __init__(self, config):
         self.acctGroupUser = getattr(config.BossAir, 'acctGroupUser', "cmsdataops")
 
         # Build a requirement string
-        self.reqStr = "stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS=?=\"any\") || (GLIDEIN_REQUIRED_OS=?=REQUIRED_OS)) && (TARGET.Cpus >= RequestCpus)"
+        self.reqStr = ('stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) '
```

Comment:
Yeah, I figured there had to be a pythonic way of doing it without continuation marks since there is an effort to reduce the number of places they are needed. Just have to be careful with spaces since they must be put in manually.